packages/browser/README.md

```diff
@@ -192,11 +192,21 @@ export default defineComponent({
 </script>
 ```
 
+## Support for Web Workers (Experimental)
+ While this package does not support web workers out of the box, there are options:
 
+1. Run analytics.js in a web worker via [partytown.io](https://partytown.builder.io/). See [our partytown example](../../examples/with-next-js/pages/partytown). **Supports both cloud and device mode destinations, but not all device mode destinations may work.**
 
-### For snippet users only: How to add typescript support
+2. Try the [Segment Public API SDK](https://github.com/segmentio/public-api-sdk-typescript). **Warning: cloud destinations only!**
```

Comment:
What is this for? Given the SDK is not relevant to data collection, does it need to be listed here?